app/docker/views/services/create/createservice.html

```diff
@@ -18,8 +18,8 @@
           <por-image-registry
             model="formValues.RegistryModel"
             auto-complete="true"
-            label-class="col-sm-1"
-            input-class="col-sm-11"
+            label-class="col-sm-2"
```

Comment:
I would have liked to use col-sm-3 col-lg-2, but the access control toggle (global component) uses col-sm-2 only, so I matched this style